### DISCUSSION TITLE
Data assimilation without CDF matching approach

### DISCUSSION THREAD

**Discussion by der-knight** [NASA-LIS/LISF#879] [General]
Hello LISF community 
I have a question regarding data assimilation in LISF
Since CDF matching leads to loss of information due to processes like irrigation. Does LISF support data assimilation without a CDF matching approach? 
If yes, is there a test case?

Regards

**Comment by dmocko** [comment]
Hi,

I believe that you are specifically asking about soil moisture data assimilation, is that correct?

Assuming so, you are correct that CDF matching is not optimal over irrigated areas.  This issue is an active area of research on the LIS team, and no test case is available.

You may find it helpful to review these journal articles:

Abolafia-Rosenzweig, R., B. Livneh, E.E. Small, and S.V. Kumar, 2019: Soil moisture data assimilation to estimate irrigation water use. Journal of Advances in Modeling Earth Systems, 11, https://doi.org/10.1029/2019MS001797

Kumar, S.V., C.D. Peters-Lidard, J.A. Santanello, R.H. Reichle, C.S. Draper, R.D. Koster, G. Nearing, and M.F. Jasinski, 2015: Evaluating the utility of satellite soil moisture retrievals over irrigated areas and the ability of land data assimilation methods to correct for unmodeled processes. Hydrol. Earth Syst. Sci., 19, 4463-4478, https://doi.org/10.5194/hess-19-4463-2015

David

**Comment by cbblanke** [comment]
I believe you can turn CDF matching off if you want.  You might be interested in our paper:
Blankenship, C. B., J. L. Case, W. L. Crosson, and B. T. Zavodsky, 2018: Correction of Forcing-Related Spatial Artifacts in a Land Surface Model by Satellite Soil Moisture Data Assimilation. IEEE Geosci. Remote Sens. Lett.,15(4), 498-502. doi: 10.1109/LGRS.2018.2805259

where we used a nonlocalized CDF matching (each pixel's corrected climatology does not have to match the model climatology for the same pixel, rather it is was applied for all points matching the vegetation type within a fairly large radius (300 km).   I had to do a custom program to create the CDF correction files.
